kythe/go/util/riegeli/riegeli.go

```diff
@@ -66,6 +66,16 @@ func BrotliCompression(level int) CompressionType {
 	return &compressionLevel{brotliCompression, level}
 }
 
+// ZSTDCompression returns a CompressionType for zstd compression with the
+// given compression level.  If level < 0 || level > 22, then the DefaultZSTDLevel
```

Comment:
What's special about 22?

Reply:
Dunno.  It's just part of the zstd spec.